scripts/lib/aws.sh

```diff
@@ -26,3 +26,22 @@ ensure_aws_k8s_tester() {
         chmod +x $TESTER_PATH
     fi
 }
+
+ensure_ecr_image() {
+  local __image_tag="$1"
```

Comment:
i know variable name might be confusing. actually currently __image_tag is "service_name:short_commit_sha". __image_tag because we are actually tagging image 
$AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$REPO_NAME by tag "service_name:short_commit_sha"
https://github.com/varun1524/aws-service-operator-k8s/blob/integtest/scripts/lib/k8s.sh#L21

I will update tag to ack-$service_name-$commit

IMAGE_NAME is this: 
https://github.com/varun1524/aws-service-operator-k8s/blob/integtest/scripts/run-integration-tests.sh#L91
That will be universal so I am pulling that from there. 

and version will be "ack-$service-$commit"

Please let me know if that looks good.

Image to show how it will look to avoid confusion.
<img width="1271" alt="Screen Shot 2020-06-25 at 11 36 53 AM" src="https://user-images.githubusercontent.com/26971990/85779255-44b31680-b6d8-11ea-960c-615d6622ecb6.png">

Reply:
ah, gotcha. yeah, maybe just rename the variable to make it clear it includes the service name :)